### PR TITLE
[s] local powergamer seen flying through space swinging crowbar wildly at void

### PR DESCRIPTION
**Pull request by Incoming5643**
Occasionally there will be a space ruin that is just a single fake tile of space. If you somehow manage to find this tile, you'll get a free pulse rifle.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -289,6 +289,13 @@
 	PoolOrNew(/obj/item/stack/tile/plasteel, src)
 	new /obj/item/weapon/crowbar(src)
 
+/obj/item/weapon/storage/backpack/satchel/flat/rngjesus/New()
```
Comment (optimumtact) [inline code/game/objects/items/weapons/storage/backpack.dm]: >can't even customise the type of object it spawns
>shit path name
>no ability to set probabilitys

bad

**Comment by Cheridan** [comment]
won't this actually not be that hard to find since the tile will stop your space drifting?

**Comment by Iamgoofball** [comment]
maps are 255x255 there are 65035 tiles on each z-level

**Comment by optimumtact** [comment]
65035 chances for this ruin to spawn.

**Comment by Incoming5643** [comment]
That's not how space ruins work

**Comment by Iamgoofball** [comment]
yeah, plus its nnot guaranteed

**Comment by Incoming5643** [comment]
Also to be clear this pull belongs to the "joke pull that might accidentally turn into real pull via input and how I feel in the morning" class of PR

**Comment by MMMiracles** [comment]
65035 potential spawn positions multiplied by the amount of z-levels that allow for ruins to spawn, then adding in the chance that it might not even spawn.

The actual odds of finding this are so minuscule that it basically doesn't exist in a round, but could you imagine the amount of ass-blasted complaints that could come from this if someone manages to find it, like an antag?

That alone should be enough for a merge.

**Comment by ike709** [comment]
Give it a 25% chance to give you a sord instead of a pulse rifle. That should make this rare enough to merge.

**Comment by Kromgar** [comment]
But why? Why even make this

**Comment by optimumtact** [comment]
for fun

**Comment by AnturK** [comment]
I'd be okay with this if ruins had importance weight. This kinda makes the system spawn one ruin less overall which is kinda bad since space ruins are rare as it is.

**Comment by optimumtact** [comment]
What antur said so then we can merge it

**Comment by lzimann** [comment]
Hey, even if it is extremely rare, can you add an admin log/message for when someone manages to find the pulse rifle?

**Comment by Incoming5643** [comment]
Joke pull transitioned to real pull. I'm not going to lay out odds to spawn for each ruin in this pull because that's kind of a completely different thing.

**Comment by optimumtact** [comment]
You need an admin message for when it happens

**Comment by Incoming5643** [comment]
There is an admin message, it's using the /prize verson of the rifle, which makes one

**Comment by optimumtact** [comment]
Oh neat

**Pull request merged**